cairo/tests/fixtures/runner.py

```diff
@@ -166,7 +192,8 @@ def _factory(entrypoint, *args, **kwargs):
 
         runner.end_run(disable_trace_padding=False)
         if request.config.getoption("proof_mode"):
-            return_data_offset = serde.get_offset(return_data.cairo_type)
+            # TODO: should we also support multiple return data types in proof mode?
```

Comment:
@ClementWalter not sure I understand what should be done in proof mode here. do I want the implicits to be included?

Reply:
proof mode needs this to include returned data into the public memory, implicit args are not meant to be different but, if only builtins, were done in the builtin final stack